test/download-test.js

```diff
@@ -259,6 +351,83 @@ describe('Download video', () => {
         });
       });
     });
+
+    describe('that should be chunked', () => {
+      it('Starts downloading video successfully and data equal to stored file', done => {
+        const scope = nock(id, {
+          type: 'regular',
+          player: true,
+        });
+
+        const dlChunkSize = 1024 * 200;
+        const stream = ytdl(id, { filter: format => !format.hasAudio, dlChunkSize });
+
+        stream.on('info', (info, format) => {
+          scope.urlReply(format.url, 206, () => fs.createReadStream(video, { start: 0, end: dlChunkSize - 1 }), {
+            'content-range': `bytes=0-${dlChunkSize - 1}/${filesize}`,
+            'content-length': dlChunkSize,
+            'accept-ranges': 'bytes',
+          });
+
+          stream.on('progress', (chunk, downloaded) => {
+            if (downloaded >= dlChunkSize) {
+              scope.urlReply(
+                format.url,
+                206,
+                () => fs.createReadStream(video, { start: dlChunkSize, end: filesize - 1 }),
+                {
+                  'content-range': `bytes=${dlChunkSize}-${filesize - 1}/${filesize}`,
+                  'content-length': filesize - downloaded,
+                  'accept-ranges': 'bytes',
+                });
+              stream.removeAllListeners('progress');
+            }
+          });
+        });
+
+        const filestream = fs.createReadStream(video);
+        streamEqual(filestream, stream, (err, equal) => {
+          assert.ifError(err);
+          scope.done();
+          assert.ok(equal);
+          done();
+        });
+      });
+    });
+
+    it('Chunks video only and chunk size matches given size', done => {
+      const dlChunkSize = 1024 * 200;
+      const stream = ytdl.downloadFromInfo(testInfo, { filter: format => !format.hasAudio, dlChunkSize });
```

Comment:
good opportunity to use `filer: 'videoonly'`

Reply:
`filer: 'videoonly'` returns `video + audio` format 😕